src/domain/Collection/BillList.js

```diff
@@ -12,14 +12,28 @@ import { useUserAuth } from 'shared/contexts/UserAuthContext';
 import { roleType } from 'data/constants';
 import { isAfter } from 'date-fns';
 
+// for btn
+import Button from '@mui/material/Button';
+
 const BillList = ({ fmk, isLoading }) => {
-  const { user: { role }} = useUserAuth();
+  const {
+    user: { role },
+  } = useUserAuth();
   const [isSave, setIsSave] = useState(false);
-  const [errorMessage, setErrorMessage] = useState("");
+  const [errorMessage, setErrorMessage] = useState('');
+
+  //create copy of billsList
+  const createBillsCopy = () => {
+    let billsList = fmk.values.bills;
+    //copy of bills data in formik values prop.
+    let billsListCopy = [...billsList];
+    return billsListCopy;
+  };
```

Comment:
You can make it simple like below
`
const createBillsCopy = () => [...fmk.values.bills];
`